ui/src/pages/Login/index.tsx

```diff
@@ -18,11 +18,14 @@ const Login: React.FC = () => {
     values.password = encryptText(values.password, publicKey) as string;
     const res = await user.login(values);
     if (res.successful) {
-      if (res.data.needReset) {
-        navigate('/reset');
-      } else {
-        navigate('/overview');
-      }
+      // Set a timer to wait for permissions to update
+      setTimeout(() => {
+        if (res.data.needReset) {
+          navigate('/reset');
+        } else {
+          navigate('/overview');
+        }
+      }, 500);
```

Comment:
Why do we need this timer to wait for permissions to be updated? What will happen if no such a timer?

Reply:
If we don't have this timer, access will not be updated after switching users with different permissions.

Reply:
Maybe I should add l `loading` to the login button.